cluster-api/src/main/java/io/aiven/klaw/clusterapi/models/enums/AclAttributes.java

```diff
@@ -0,0 +1,13 @@
+package io.aiven.klaw.clusterapi.models.enums;
+
+public enum AclAttributes {
```

Comment:
If I may suggest something. If the string value is supposed to always match (lowercase) the enum value, then you could also write something like
```java
public enum AclAttributes {
  TOPIC, PERMISSION, USERNAME;

  public String toLowerCase() {
        return name().toLowerCase();
    }
}
```

If you want or need more flexibility, you may also emulate extensible enums with interfaces (please use better names).
```java
interface Stringify {
  String stringify();
}

public enum AclAttributes implements Stringify {
  TOPIC, PERMISSION, USERNAME;

  @Override
  public String stringify() {
        return name().toLowerCase();
    }
}
```

Reply:
@sekaiser prefer to not apply EnumMap in this situation especially when the rest call needs lower case keys and it involves more method calls, and then I don't see that efficiency.

Reply:
Also, I just noticed the enum `KafkaSupportedProtocol`. So on a second thought my suggestion requires much more effort than is probably needed and how much does it actually improve the efficiency in this case and does it really matter? Same applies to the MapSerializer.
Sorry for the distraction.